### ISSUE TITLE
🚧 🐞 Bug Report: Render term descriptions with wpautop, same as in live editor

### ISSUE THREAD

**Issue by ADAGEBF** [elementor/elementor#20674]
### Prerequisites

- [X] I have searched for similar issues in both open and closed tickets and cannot find a duplicate.
- [X] The issue still exists against the latest stable version of Elementor.

### Description

When editing an archive template, the description is rendered in text widgets with **wpautop** function. 
But when publishing, the final frontend is rendered without **wpautop**.

### Steps to reproduce

1. On a new site with Hello, Elementor and Elementor Pro, 
2. Edit a category description with a text including several line breaks. 
3. Create an archive template applied for categories, and add a text widget.
4. Use the dynamic "term description", and notice the line breaks are rendered well (with **wpautop**).
5. Publish your work, then go frontend and notice the line breaks are not rendered anymore. 

### Isolating the problem

- [X] This bug happens with only Elementor plugin active (and Elementor Pro).
- [X] This bug happens with a Blank WordPress theme active ([Hello theme](https://wordpress.org/themes/hello-elementor/)).
- [X] I can reproduce this bug consistently using the steps above.

### System Info

Hello
Elementor
Elementor Pro

**Comment by nicholaszein** [comment]
Hey @ADAGEBF, 

Thank you for submitting your report! 🙏 

### ❌ While reviewing your report, we noticed it was missing some key pieces of information:

- **Prerequisites:** please search the repository before submitting an issue;
- **Description:** We understand what you're talking about, but it wasn't clear what widget were you using, or if you were using the dynamic-tag feature or not. Please explain exactly where the issue happens and in which widgets;
- **System info:** You did not provide any information about your [System Info](https://elementor.com/help/systems-info/) as outlined in our [guidelines](https://github.com/elementor/elementor/issues/new?assignees=&labels=status%2Fawaiting_triage&template=bug.yml). We need the complete report to be able to accurately reproduce your issue.

Without them, we simply can't help. 😕

Best regards,

Nick

**Comment by nicholaszein** [comment]
Hello @ADAGEBF,

any update on the missing information? We really need it in order to be able to validate and reproduce your issue.

Please provide the information requested.

Kind regards

**Comment by nicholaszein** [comment]
Hello @ADAGEBF, 

#### 🕙 It's been a while since we answered you, and we haven't received any feedback. 

## 🚧 For this reason, we are closing this issue because of no activity.

👉 If you wish to continue this discussion, please follow the guidelines for opening a new [🐞 Bug Report](https://github.com/elementor/elementor/issues/new?assignees=&labels=status%2Fawaiting_triage&template=bug.yml).

Feel free to submit a new issue in accordance with these guidelines. 😉 

Kind regards
